plugins/BEdita/Core/src/Model/Table/AsyncJobsTable.php

```diff
@@ -268,6 +268,21 @@ protected function findCompleted(Query $query)
     }
 
     /**
+     * Finder for uncompleted async jobs.
+     *
+     * This finder returns a query object that filters asynchronous jobs that haven't been completed yet.
+     *
+     * @param \Cake\ORM\Query $query Query object instance.
+     * @return \Cake\ORM\Query
+     */
+    protected function findUncompleted(Query $query)
```

Comment:
I don't like much the name. 😄  `findIncomplete` ?

However, should this finder find _all_ jobs that haven't been marked as completed? Results will contain failed and expired jobs too.

For the purpose of this PR, I think that a `findPending` finder would fit better: find only jobs that are scheduled to be run in the future, i.e.:
```sql
SELECT * FROM async_jobs AS AsyncJobs
    WHERE AsyncJobs.scheduled_from > NOW() AND /* maybe? */ AsyncJobs.completed IS NULL:
```